tests/Integration/ScriptsTest.php

```diff
@@ -331,27 +331,27 @@ public function test_enqueue_js_api_with_secret(): void {
 	 * @group scripts
 	 * @group settings
 	 */
-	public function test_do_not_track_logged_in_users(): void {
+	public function test_track_logged_in_users(): void {
 		TestCase::set_options(
 			array(
 				'api_secret'                => 'hunter2',
-				'track_authenticated_users' => false,
+				'track_authenticated_users' => true,
 			)
 		);
 		$new_user_id = $this->create_test_user( 'bill_brasky' );
 		wp_set_current_user( $new_user_id );
-
+		$this->go_to_new_post();
```

Comment:
Test was passing for all cases so added this line to make test work as expected.